### PR TITLE
Backport 4.3: azure payg jcayouette #23134

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/23134 ; **there at the beginning in the description find a link to the rendered output.**

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,14 @@
...
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+<<<<<<< HEAD
+=======
+- Added Pay-as-you-go for Azure documentation to the Specialized Guides
+  book
+- Removed traditional client stack documentation such as rhnsd in
+  Troubleshooting Guide
+- Updated Red Hat OVAL data URL and file in CentOS Client Configution Guide
+>>>>>>> de207e6b6 (Azure payg jcayouette (#2645))
```
Comment (keichwa) [inline .changelog:14]:
```suggestion
- Added Pay-as-you-go for Azure documentation to the Specialized Guides
  book
- Updated Red Hat OVAL data URL and file in CentOS Client Configution Guide
```

File: .changelog [hunk 2]
```diff
@@ -4,6 +4,9 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Added Pay-as-you-go for Azure documentation to the Specialized Guides
+  book
+- Updated Red Hat OVAL data URL and file in CentOS Client Configution Guide
```
Comment (keichwa) [inline .changelog:9]:
```suggestion
```

**Comment by keichwa** [comment]
@aosthof, I cannot set you as a reviewer in this project (uyuni-docs).  If possible, can you add yourself (it often works if you click the "Files changed" tab and then start as usual).

**Comment by 0rnela** [comment]
@ke - Approved (subject to the PR being 1:1 replica of the master branch PR).

**Review by 0rnela — APPROVED** [review]

**Review by aosthof — APPROVED** [review]
LGTM - thanks for getting this done in a timely fashion, really appreciated!

**Review by mcalmer — APPROVED** [review]
LGTM

**Comment by keichwa** [comment]
> @ke - Approved (subject to the PR being 1:1 replica of the master branch PR).

Yes, it should basically.  It was a cherry-pick plus very minor improvements with the rendering.

**Pull request merged**